proposals/lip-0052.md

```diff
@@ -151,39 +200,61 @@ The function `uint64be(x)` returns the big endian uint64 serialization of an int
 
 In this LIP, `length(byteSequence)` returns the length in bytes of `byteSequence`.
 
-#### Functions from Other Modules
+#### Logic from Other Modules
+
+Calling a function `fct` from the [Interoperability module][lip-0045] is represented by `Interoperability.fct(required inputs)`.
+
+### NFT Identification
+
+All NFTs in the ecosystem are identified by using the following three values:
+
+- `chainID`, always the chain ID of the chain that created the NFT,
+- `collection`, an integer specified at NFT creation,
+- `index`, assigned at NFT creation to the next available index in the collection.
+
+In this LIP, the NFT identifier `nftID` is a 16 bytes long concatenation of the `chainID` of the NFT native chain and the serializations of `collection` and `index`: `nftID = chainID + uint32be(collection) + uint64be(index)`. This is for example used in all input formats for the module's exposed logics. This allows the exposed logic interfaces to be simple and uniform.
+
+#### NFT ID and Native NFT
```

Comment:
so for instance this section could be removed completely (see previous comment about aliasing native NFTs)